### DISCUSSION TITLE
Dark mode Issue

### DISCUSSION THREAD

**Discussion by TemkinTegegn** [sumatrapdfreader/sumatrapdf#3876] [Q&A]
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/77839113/c6325d88-387c-4f9b-b3d6-ddcec58563f0)
As you can see in this screenshot. I selected dark mode but the reading background is still white and when I switch it to light mode, the reading background background will be dark. So my question is how can I make the reading background dark in dark mode and why it can make it dark automatically when I switch it to dark mode by itself? @everyone

**Comment by Forbidden-Creater** [comment]
I have the same problem. And it seems that the dark mode uses a fixed color scheme built into the program. Because I have changed the background color value in "SumatraPDF-settings.txt", but when I choose the dark theme, the background color turns into white.

**Reply by tihomiho** [reply]
Also, the icons on the toolbar are getting fuzzy

**Reply by TemkinTegegn** [reply]
Set invertedcolors = true and you will get complete dark mode. I get the reply from the one of the developers but it is still has some issues. No matter what color you chose in TextColor and BackgroundColor, it doesn't have any effect at all!

**Comment by tihomiho** [comment]
![2023-11-17_082132](https://github.com/sumatrapdfreader/sumatrapdf/assets/20873648/e14f91d8-f2a9-4da4-ae82-f1cdb957cb79)

**Reply by TemkinTegegn** [reply]
I forwarded this issue to one of the developers as well.

**Reply by Av1ators** [reply]
+1

**Comment by Mycroft-47** [comment]
Hey everyone, a quick observation. The BackgroundColor setting (#EDD1B0 in my case) isn't taking effect with the new dark theme. Is there a way we can make it work together?
![BackgroundColor-Issue-DarkTheme png](https://github.com/sumatrapdfreader/sumatrapdf/assets/118926775/b3a1728e-bd52-4281-a90e-a9575f181e59)

**Comment by GitHubRulesOK** [comment]
@Mycroft-47 Perhaps your not using the updated version
I have inverted it so its down under, oddly it looks like USA! , but then again inversion is opposite to natural or is it ?

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/bb177119-12aa-4b1d-8655-cb8ec5bf18ea)

**Reply by Mycroft-47** [reply]
Unfortunately, I'm already using the latest available release, [3.5.2](https://github.com/sumatrapdfreader/sumatrapdf/releases/tag/3.5.2rel)
![3.5.2](https://github.com/sumatrapdfreader/sumatrapdf/assets/118926775/04de8a3e-8d60-423c-bf64-ae056a0e5a8a)

**Reply by GitHubRulesOK** [reply]
[https://www.sumatrapdfreader.org/prerelease](https://www.sumatrapdfreader.org/prerelease)

**Reply by Mycroft-47** [reply]
thanks

**Comment by Av1ators** [comment]
the icons in excuse/black mode are grainy and ruined. is a fix possible?

**Comment by kjk** [comment]
The icons in dark themes have been improved in latest pre-release https://www.sumatrapdfreader.org/prerelease

**Comment by MathisP75** [comment]
Unfortunately, there are still a couple issues with dark mode.

1. The toolbar separators look way too thick compared to the light theme
<img width="461" alt="image" src="https://github.com/user-attachments/assets/27244961-b8f9-43b1-b79d-6c983116c624">

2. There is no visual distinction between the toolbar/header and the view background, like in light mode, where there are different shades of white.

3. When in full screen, there is a white line at the bottom of the app
![image](https://github.com/user-attachments/assets/15e0f5d5-853a-4c0a-8f76-772c20f49668)

4. Arrows are black in drop down menus
![image](https://github.com/user-attachments/assets/c40f3cb0-51a5-4301-bbae-5de068b4d35f)

5. The scrollbars are white
